frontend/src/modules/SimulationTimeSeries/view/hooks/useMakeViewStatusWriterMessages.ts

```diff
@@ -5,11 +5,17 @@ import { EnsembleSetAtom } from "@framework/GlobalAtoms";
 import type { ViewContext } from "@framework/ModuleContext";
 import type { RegularEnsemble } from "@framework/RegularEnsemble";
 import type { ViewStatusWriter } from "@framework/StatusWriter";
+import { usePropagateAllApiErrorsToStatusWriter } from "@modules/_shared/hooks/usePropagateApiErrorToStatusWriter";
 import type { Interfaces } from "@modules/SimulationTimeSeries/interfaces";
 
 import { showObservationsAtom } from "../atoms/baseAtoms";
-import { queryIsFetchingAtom, realizationsQueryHasErrorAtom, statisticsQueryHasErrorAtom } from "../atoms/derivedAtoms";
-import { vectorObservationsQueriesAtom, regularEnsembleHistoricalVectorDataQueriesAtom } from "../atoms/queryAtoms";
+import { queryIsFetchingAtom } from "../atoms/derivedAtoms";
+import {
+    vectorObservationsQueriesAtom,
+    regularEnsembleHistoricalVectorDataQueriesAtom,
+    vectorDataQueriesAtom,
+    vectorStatisticsQueriesAtom,
+} from "../atoms/queryAtoms";
 
 export function useMakeViewStatusWriterMessages(
     viewContext: ViewContext<Interfaces>,
```

Comment:
I know this was not introduced in your PR, but `viewContext` seems to be unused. Thus we should remove it.